resolwe_bio/tools/cuffnorm_output_table_headers.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python2
+# pylint: disable=missing-docstring,invalid-name
+# XXX: Refactor to a comand line tool and remove pylint disable
+"""Rename cuffnorm output tables headers."""
+from __future__ import absolute_import, division, print_function
+
+import argparse
+from operator import itemgetter
+
+parser = argparse.ArgumentParser(description="Rename cuffnorm output tables headers")
+parser.add_argument('files', help="All .cxb files")
+parser.add_argument('replicates', help="Define replicates and groups")
+parser.add_argument('sample_names', help="Sample names")
+args = parser.parse_args()
+
+files = args.files.split(",")
+replicates = args.replicates.split(",")
+sample_names = args.sample_names.split(",")
+
+samples=[]
+for x in sorted(zip(files,replicates,sample_names), key=itemgetter(1)):
+    samples.append(x[2])
+                                                    
+def readwrite(table_name,sample_list):
+    file_lines=[]
+    with open(table_name, 'r') as genes_table:
+        genes_table.next()
+        for line in genes_table:
+            file_lines.append(line)
+
+    with open(table_name, 'w') as genes_table:
+        genes_table.write("tracking_id"+'\t')
+        genes_table.write('\t'.join(sample_list) + '\n')
+        for line in file_lines:
+            genes_table.write(line)
+
+tables=['cds.count_table','cds.fpkm_table','genes.count_table','genes.fpkm_table',
```

Comment:
put each file name in separate line:

    tables=[
        'cds.count_table',
        'cds.fpkm_table',
        'genes.count_table',
        'genes.fpkm_table',
        'isoforms.count_table',
        'isoforms.fpkm_table',
        'tss_groups.count_table',
        'tss_groups.fpkm_table',
    ]